src/embed-utils/oembed.ts

```diff
@@ -1,5 +1,5 @@
 export const getScriptSrcFromOembedHTML = (html: string): string | null => {
-  const src = html.match(/script async src\s*=\s*["'](.+?)["']/);
+  const src = html.match(/script(.*)src\s*=\s*["'](.+?)["']/);
```

Comment:
```suggestion
  const src = html.match(/script(?:.*)src\s*=\s*["'](.+?)["']/);
```

Dann brauchst du die Zeile unten nicht anpassen